benchmark/redisgraph/scripts/retrieve_graph500_rdb_from_s3.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Should this and the other S3-accessing scripts be part of this repo?

Reply:
they are only accessing public content on s3. Do you think we should remove them?

Reply:
No, if the buckets are public then they're fine! I was just unsure.